iroh-sync/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+//! Multi-dimensional key-value documents with an efficient synchronization protocol
+//!
+//! The crate operates on [Replicas](Replica). A replica contains an unlimited number of
+//! [Entrys][Entry]. Each entry is identified by a key, its author, and the replica's
```

Comment:
```suggestion
//! [Entrys](Entry). Each entry is identified by a key, its author, and the replica's
```
?

But if cargo-doc is happy then it's fine i guess...